api/api_client.go

```
@@ -62,18 +63,37 @@ func SendPut(targetUrl string, header map[string]string, param map[string]string
+		return []byte(""), err
+	}
+	if _, err := io.Copy(part, fileContent); err != nil {
+		return []byte(""), err
+	}
+	if err := writer.Close(); err != nil {
+		return []byte(""), err
 	}
 
-	httpClient := &http.Client{}
-	request, err := http.NewRequest(method, baseUrl+targetUrl, bytes.NewBuffer(body))
+	request, err := http.NewRequest("POST", baseUrl+withQuery(targetUrl, param), body)
```

golangci-lint (2.12.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)